selector.h

```diff
@@ -8,6 +8,8 @@ class Selector {
         bool first = true; 
         int up_key_state = 0;
         int down_key_state = 0;
+        int left_key_state = 0;
+        int right_key_state = 0;
```

Comment:
Seems a lot of these variables registering key-states are now unused, consider cleaning up.

Reply:
Thanks Riley, forgot about those, have deleted now.